### PR TITLE
Issue60: Hacktoberfest celebration: Blogs: 'Rooster Rescue' by Alexis…

### PR DESCRIPTION
**Pull request by LittleMissButtons**
## Description

New blog submission for 'Rooster Rescue' by LittleMissButtons (Alexis Pendleton).

## Related Issue

Issue 60: Hacktoberfest celebration: Blogs: 

## Acceptance Criteria

Write a blog on anything related to pet or stray animals: Hacktoberfest

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
| ✓ | :scroll: Docs              |

### Before

image

### After

image
image

- Blog12 card added to the bottom of the 'Blog' page
- Rooster Rescue blog page added
- b12-poster image added for that blog
- Writer added to the Contributions.md

## Testing Steps / QA Criteria

Blog page and new blog at the bottom on the page. Click 'Rooster Rescue' card for the full blog.

### CODE REVIEW

File: blogs/alexis.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                    <div class="text-gray-700 text-xl font-normal my-0 mx-[1rem] tracking-normal">
+                        <a href="https://littlemissbuttons.github.io">
+                            <h3 class="uppercase"><b>Alexis Pendleton</b></h3>
+                            <p>October 27, 2022</p>
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="content">
+                <p class="text-[#5b5963] my-[1rem] mx-0 text-xl">‘Has anyone lost a chicken? We’ve one at the shop!’
```
Comment (Anuran12) [inline blogs/alexis.html:48]:
```suggestion
                <h3 class="text-xl xl:text-2xl font-semibold tracking-wide">‘Has anyone lost a chicken? We’ve one at the shop!’</h3>
```

Do this changes in all your heading points @LittleMissButtons

**Review by Anuran12 — CHANGES_REQUESTED** [review]

File: blogs/alexis.html (hunk 1 above)
Comment (Anuran12) [reply blogs/alexis.html:48]: @LittleMissButtons  Otherwise its all okay.. please quickly change this.

Comment (LittleMissButtons) [reply blogs/alexis.html]: Hi, thanks for the feedback and I have updated that now and pushed the changes.

Hopefully that comes through ok. Please could you add the Hacktoberfest-Accepted label to this PR. Cheers

**Review by Anuran12 — APPROVED** [review]
Look good to me

**Review by akshitagupta15june — APPROVED** [review]
thanks @LittleMissButtons and @Anuran12

**Comment by Anuran12** [comment]
@akshitagupta15june  I am making another PR within few minutes

**Comment by akshitagupta15june** [comment]
@Anuran12 can you look at the discord dm , i just dmed you regarding something

**Pull request merged**